restapi/stories.go

```diff
@@ -57,12 +73,74 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 
 	for i := 0; i < len(stories); i++ {
 		city := stories[i].CurrentCity
-		coordinates, err := api.locationFinder.GetCityCenter(city)
+		stories[i].CurrentCity = strings.Title(strings.ToLower(city))
```

Comment:
Can we move this logic out into a function? I would like the create stories endpoint to be clean

Reply:
moved it all into a function